### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by gomoripeti**
(I am afraid I still cannot push to this repo directly, so submitting PR)

### CODE REVIEW

File: src/typechecker.erl [hunk 1]
```diff
@@ -1436,6 +1435,16 @@ glb_types({type, _, N, Args1}, {type, _, N, Args2}) ->
...
 
+get_rec_field_type(FieldName,
+                   [{typed_record_field,
+                     {record_field, _,
+                      {atom, _, FieldName}, _}, Ty}|_]) ->
+    Ty;
+get_rec_field_type(FieldName, [_|RecFieldTypes]) ->
+    get_rec_field_type(FieldName, RecFieldTypes);
+get_rec_field_type(_, []) ->
+    {error, not_found}.
```
Comment (josefs) [inline src/typechecker.erl]: I would suggest throwing an exception here instead.

**Comment by josefs** [comment]
Not sure why you can't push directly to the repo. Github says "Awaiting gomoropeti's resonse". What happens if you go to https://github.com/josefs/Gradualizer/invitations ?

File: src/typechecker.erl (hunk 1 above)
Comment (gomoripeti) [reply src/typechecker.erl]: 👍 pushed a change to throw an exception.

**Comment by gomoripeti** [comment]
I thought it was only a notification, but I indeed had to push the button. Now I can happily push, thank you :)

**Pull request merged**